### ISSUE TITLE
What is the 'retrieve' method for?

### ISSUE THREAD

**Issue by confile** [metafizzy/infinite-scroll#366]
Someone posted me the following:

"Hi,
I'm also looking fo a way to achieve the functionality of allowing a user to jump back quickly to where they were in a long list of infinite scroll content.
I see it as being incredibly useful as otherwise it is very frustrating to lose your place in a long list and have to scroll back through everything to get back to where you were.

I came accross the 'retrieve' method in the source code, which looked like it would help with this.

Essentially I am tracking the "page" number that a user was on, as well as the ID of the object they clicked on (I've got an Isotope / Masonry set up on this site) so that if they hit the back button in their browser, they will be brought back to exactly where they were in the long list.

It's working perfectly when there are only 2 'pages' of infinite scroll content, but not when there are any more than 2 pages.

Basically, on page load I check for the page number and ID variables and I use the page number variable (pg in the code below) to invoke the retrieve method:

$('#isoContainer').infinitescroll('retrieve',pg);

Is this not what the 'retrieve' method is meant for? Would really appreciate some clarification on how this works and why it only seems to work when retrieving page 2, and not 3,4, or 5 ets...

Thanks :)
"

That would interest me too. What does the retrieve method do?
